app/controllers/miq_policy_controller.rb

```diff
@@ -380,6 +380,102 @@ def replace_right_cell(options = {})
+      case @sb[:nodeid]
+      when 'host'
+        summary = _("All Host Compliance Policies")
+        adding = _("Adding a new Host Compliance Policy")
+        editing = @edit ? _("Editing Host Compliance Policy \"%{name}\"") : _("Host Compliance Policy \"%{name}\"")
+      when 'vm'
+        summary = _("All VM and Instance Compliance Policies")
+        adding = _("Adding a new VM and Instance Compliance Policy")
+        editing = @edit ? _("Editing VM and Instance Compliance Policy \"%{name}\"") : _("VM and Instance Compliance Policy \"%{name}\"")
+      when 'containerReplicator'
+        summary = _("All Container Replicator Compliance Policies")
+        adding = _("Adding a new Container Replicator Compliance Policy")
+        editing = @edit ? _("Editing Container Replicator Compliance Policy \"%{name}\"") : _("Container Replicator Compliance Policy \"%{name}\"")
+      when 'containerGroup'
+        summary = _("All Container Pod Compliance Policies")
+        adding = _("Adding a new Container Pod Compliance Policy")
+        editing = @edit ? _("Editing Container Pod Compliance Policy \"%{name}\"") : _("Container Pod Compliance Policy \"%{name}\"")
+      when 'containerNode'
+        summary = _("All Container Node Compliance Policies")
+        adding = _("Adding a new Container Node Compliance Policy")
+        editing = @edit ? _("Editing Container Node Compliance Policy \"%{name}\"") : _("Container Node Compliance Policy \"%{name}\"")
+      when 'containerImage'
+        summary = _("All Container Image Compliance Policies")
+        adding = _("Adding a new Container Image Compliance Policy")
+        editing = @edit ? _("Editing Container Image Compliance Policy \"%{name}\"") : _("Container Image Compliance Policy \"%{name}\"")
+      when 'containerProject'
+        summary = _("All Container Project Compliance Policies")
+        adding = _("Adding a new Container Project Compliance Policy")
+        editing = @edit ? _("Editing Container Project Compliance Policy \"%{name}\"") : _("Container Project Compliance Policy \"%{name}\"")
+      when 'extManagementSystem'
+        summary = _("All Provider Compliance Policies")
+        adding = _("Adding a new Provider Compliance Policy")
+        editing = @edit ? _("Editing Provider Compliance Policy \"%{name}\"") : _("Provider Compliance Policy \"%{name}\"")
+      when 'physicalServer'
+        summary = _("All Physical Server Compliance Policies")
+        adding = _("Adding a new Physical Server Compliance Policy")
+        editing = @edit ? _("Editing Physical Server Compliance Policy \"%{name}\"") : _("Physical Server Compliance Policy \"%{name}\"")
+      else
+        ui_lookup(:model => @sb[:nodeid].try(:camelize))
+        summary = _("All %{model} Compliance Policies")
+        adding = _("Adding a new %{model} Compliance Policy")
+        editing = @edit ? _("Editing %{model} Compliance Policy \"%{name}\"") : _("%{model} Compliance Policy \"%{name}\"")
```

Comment:
You also need to expand the `%{model}` token in the above 3 strings (since it's not being expanded anywhere later).

Reply:
should be working correctly now